v4-client-js/src/clients/constants.ts

```diff
@@ -43,6 +43,39 @@ export enum NetworkId {
 export const NETWORK_ID_MAINNET: string | null = null;
 export const NETWORK_ID_TESTNET: string = 'dydxprotocol-testnet';
 
+// ------------ MsgType URLs ------------
+// Default CosmosSDK
+// x/bank
+export const TYPE_URL_MSG_SEND = '/cosmos.bank.v1beta1.MsgSend';
+
+// x/gov
+export const TYPE_URL_MSG_SUBMIT_PROPOSAL = '/cosmos.gov.v1.MsgSubmitProposal';
+
+// dYdX Specific
+// x/clob
+export const TYPE_URL_MSG_PLACE_ORDER = '/dydxprotocol.clob.MsgPlaceOrder';
+export const TYPE_URL_MSG_CANCEL_ORDER = '/dydxprotocol.clob.MsgCancelOrder';
+export const TYPE_URL_MSG_CREATE_CLOB_PAIR = '/dydxprotocol.clob.MsgCreateClobPair';
+export const TYPE_URL_MSG_UPDATE_CLOB_PAIR = '/dydxprotocol.clob.MsgUpdateClobPair';
+
+// x/delaymsg
+export const TYPE_URL_MSG_DELAY_MESSAGE = '/dydxprotocol.delaymsg.MsgDelayMessage';
+
+// x/perpetuals
+export const TYPE_URL_MSG_CREATE_PERPETUAL = '/dydxprotocol.perpetuals.MsgCreatePerpetual';
+
+// x/prices
+export const TYPE_URL_MSG_CREATE_ORACLE_MARKET = '/dydxprotocol.prices.MsgCreateOracleMarket';
+
+// x/sending
+export const TYPE_URL_MSG_CREATE_TRANSFER = '/dydxprotocol.sending.MsgCreateTransfer';
+export const TYPE_URL_MSG_WITHDRAW_FROM_SUBACCOUNT = '/dydxprotocol.sending.MsgWithdrawFromSubaccount';
+export const TYPE_URL_MSG_DEPOSIT_TO_SUBACCOUNT = '/dydxprotocol.sending.MsgDepositToSubaccount';
+
+// ------------ Chain Constants ------------
+export const GOV_MODULE_ADDRESS = 'dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky';
```

Comment:
Are these two addresses the same across mainnet, testnet, staging, and dev?

Reply:
yessir, these are the same across different deployments